ci/pipeline/ray_coverage.py

```diff
@@ -0,0 +1,69 @@
+import click
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+
+COVERAGE_FILE_NAME = "ray_release.cov"
+
+
+@click.command()
+@click.argument("test_target", required=True, type=str)
+def main(test_target: str) -> None:
+    """
+    This script collects dynamic coverage data for the test target, and upload the
+    results to database (S3).
+    """
+    logger = _get_logger()
+    logger.info(f"Collecting coverage for test target: {test_target}")
+    coverage_file = os.path.join(tempfile.gettempdir(), COVERAGE_FILE_NAME)
```

Comment:
is there going to be anything the deletes this tempdir?

Reply:
Good point, I think I should use /tmp/artifacts which are setup with auto clean-up